packages/ui/src/molecules/SlideOver/stories/SlideOver.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+
+import Component from '..';
+import { Button } from '../../../';
+import mdx from './SlideOver.mdx';
+
+import type { Story, Meta } from '@storybook/react'
+
+const DefaultTemplate: Story = () => {
+  const [isOpen, setIsOpen] = useState(false)
+  const handleClose = () => setIsOpen(false)
+
+  return (
+    <>
+      <Button onClick={() => setIsOpen(!isOpen)}>Open Slide Over</Button>
+
+      <Component
+        size="partial"
+        direction="rightSide"
+        fade="in"
+        isOpen={isOpen}
+        onDismiss={handleClose}
+      >
+        <Button onClick={handleClose}>Close Slide Over</Button>
+      </Component>
+    </>
+  )
+}
+
+export const Default = DefaultTemplate.bind({})
+
+export default {
+  title: 'Molecules/SlideOver',
+  parameters: {
```

Comment:
oh! We can also list the props here in the control ;) 
<img width="827" alt="image" src="https://user-images.githubusercontent.com/3356699/194304858-83a1fb02-e680-461c-96f7-2c80a9b5c2f0.png">